generic/tests/netperf.py

```diff
@@ -148,8 +150,19 @@ def _pin_vm_threads(vm, node):
     vm.verify_alive()
     login_timeout = int(params.get("login_timeout", 360))
 
-    session = vm.wait_for_login(timeout=login_timeout)
+    server_ip = vm.wait_for_get_address(0, timeout=5)
+    if len(params.get("nics", "").split()) > 1:
+        vm.wait_for_serial_login(timeout=10, restart_network=True)
+        server_ctl = vm.wait_for_login(nic_index=1, timeout=login_timeout)
+        server_ctl_ip = vm.wait_for_get_address(1, timeout=5)
+        session = vm.wait_for_login(nic_index=1, timeout=30)
+    else:
+        server_ctl = vm.wait_for_login(timeout=login_timeout)
+        server_ctl_ip = server_ip
+        session = vm.wait_for_login(timeout=login_timeout)
+
     mac = vm.get_mac_address(0)
+    macvtap_tap = vm.get_ifname(0)
```

Comment:
it seems this variable is never used.

Reply:
it used in line 136 "target_ifaces.extend([params.get("netdst_nic1"), macvtap_tap])"

Reply:
sorry, my mistake.

Reply:
L160 and L162, the same as above.

Reply:
The one is more server, the other one is server_ctl.

Reply:
If so, please reference `vm.get_ifname(0)` directly at line 136.